draft-ietf-rtcweb-jsep.xml

```diff
@@ -3540,12 +3540,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>tls-id value, which MUST be set according to
               <xref target="I-D.ietf-mmusic-dtls-sdp" />, Section 5. If
-              this is a re-offer and the tls-id value is different
+              this is a re-offer or a response to a re-offer
+              and the tls-id value is different
```

Comment:
If the remote DTLS id changes, and the local side is not performing an ICE restart, what then?

Reply:
That's apparently forbidden:
   When a new DTLS association is established over an unordered
   transport, in order to disambiguate any packets associated with the
   newly established DTLS association, at least one of the endpoints
   MUST allocate a completely new set of ICE candidates which were not
   recently used for any other DTLS association.  This means the
   answerer cannot initiate a new DTLS association unless the offerer
   initiated ICE restart [I-D.ietf-ice-rfc5245bis].  If the answerer
   wants to initiate a new DTLS association, it needs to initiate an ICE
   restart and a new offer/answer exchange on its own.  However, an ICE
   restart does not by default require a new DTLS association to be
   established.

Reply:
So the answerer can change the tls-id, even if the offerer did not, but only if ICE is already restarting.

Seems kind of strange to permit this corner case; do you know what the thinking is here?

Reply:
No, I do not.

Reply:
OK. I would suggest that we should instead explode if the TLS id changes on the remote side during an ICE restart and the offerer didn't change it.

Reply:
so note sure I care about the following case but consider it ... the browser is using webrtc to connect to a farm of servers that are all streaming the music on hold or a football game or whatever ... session is set up and browser does an ICE restart as the wifi interface just came up on mobile device browser is running on. The offer does not change tls-id. Now the call lands on a new server and it needs new TLS session. The answer replies with a new tls-id. Seems like this should be OK